exiftool.go

```diff
@@ -4,53 +4,96 @@ import (
 	"bufio"
 	"bytes"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"io"
 	"os"
 	"os/exec"
 	"sync"
-)
 
-var binary = "exiftool"
-var executeArg = "-execute"
-var initArgs = []string{"-stay_open", "True", "-@", "-", "-common_args"}
-var extractArgs = []string{"-j"}
-var closeArgs = []string{"-stay_open", "False", executeArg}
-var readyToken = []byte("{ready}\n")
-var readyTokenLen = len(readyToken)
+	"github.com/pkg/errors"
```

Comment:
Why replacing built-in `errors` package by this one ?

Reply:
I've reverted to the built in `errors` package